LICENSE

```diff
@@ -0,0 +1,15 @@
+Apache Software License 2.0
```

Comment:
you don't really need to add this file, it's already specified in poetry https://github.com/Sceptre/jinja-unquote-resolvers-filter/blob/master/pyproject.toml#L11